cpp/CMakeLists.txt

```diff
@@ -140,6 +144,7 @@ add_library (bond
     STATIC
     ${schemas}
     ${generated_files_types}
+    ${precompiled_sources}
```

Comment:
The new precompiled sources will also need to be installed when BOND_LIBRARIES_INSTALL_CPP is defined. (This is used when we produce C++ packages for some odd build systems used internally at Microsoft.)

The description of the CMake variable `BOND_LIBRARIES_INSTALL_CPP` in Config.cmake also needs to be updated:

> If TRUE, the .cpp files for the Bond libraries will be installed under src/ as part of the INSTALL target.

Reply:
Updated.